### ISSUE TITLE
Error: Not compatible with requested type

### ISSUE THREAD

**Issue by hsmith9002** [YuLab-SMU/clusterProfiler#221]
### Prerequisites

+ [x] Have you read [Feedback](https://guangchuangyu.github.io/clusterprofiler/#feedback) and follow the [guide](https://guangchuangyu.github.io/2016/07/how-to-bug-author/)?
	* [x] make sure your are using the latest release version
	* [x] read the [documents](https://guangchuangyu.github.io/clusterprofiler/documentation/)
	* [x] google your quesion/issue

### Describe you issue

* [x] Make a reproducible example (*e.g.* [1](https://gist.github.com/talonsensei/e1fad082657054207f249ec98f0920eb))
* [x] your code should contain comments to describe the problem (*e.g.* what expected and actually happened?)

### Ask in right place

* [x] for bugs or feature requests, post here (github issue)
* [x] for questions, please post to [Bioconductor](https://support.bioconductor.org/) or [Biostars](https://www.biostars.org/) with tag `clusterProfiler`

Hello,

I am trying to generate a gene-concept netowrk from my kegg results, but I keep getting this error:

Error in pathAttr(data, length(unique(data$group))) : 
  Not compatible with requested type: [type=character; target=integer].

I also get this error when I run your example code, which makes me think I am missing a dependency or something of that kind?

Thank you
Harry

**Comment by hsmith9002** [comment]
Nevermind. I figured it out. I needed to install both the gdal and gfortran libraries so that some other packages could compile when installing clusterProfiler. ClusterProfiler does install successfully without these packages. It might be nice in the vignette to mention that at least the cnetplot function (which is an awesome plotting function btw) requires these packages to work correctly. 

Thank you
Harry
